api/v1alpha1/tracing_types.go

```diff
@@ -114,3 +116,20 @@ type RequestHeaderCustomTag struct {
 	// +optional
 	DefaultValue *string `json:"defaultValue,omitempty"`
 }
+
+// ZipkinTracingProvider defines the Zipkin tracing provider configuration.
+type ZipkinTracingProvider struct {
+	// CollectorHostname defines the hostname to use when sending spans
+	// to the Zipkin collector endpoint.
+	// +optional
+	CollectorHostname *string `json:"collectorHostname,omitempty"`
```

Comment:
Is this necessary for first iterate? There's a discussion about where's the right place for authoriy, here or under `BackendRef`

Reply:
It's not necessary but I lacked the context on the other discussion. Happy to pull this out and wait for the outcome of that discussion

Reply:
The standard Zipkin ecosystem does nothing with this.